gonum/Dockerfile

```diff
@@ -36,5 +36,6 @@ ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
 ENV CGO_LDFLAGS "-lopenblas"
 
 RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
+RUN go get -u gonum.org/v1/gonum/... && go get -d gonum.org/v1/netlib/...
```

Comment:
Why different "-u" and "-d" are chosen here?
You don't want to install the latter "gonum.org/v1/netlib" packages?

Reply:
-u means **Update** to newer minor version dependency and -d means **Download** dependency but not build, which come from gonum official installation instructions,
- [https://github.com/gonum/gonum]
- [https://github.com/gonum/netlib]
 Given "gonum.org/v1/gonum" is core package, if fail to install, there is no need install "gonum.org/v1/netlib"